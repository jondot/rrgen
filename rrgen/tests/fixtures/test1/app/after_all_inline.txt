// Both structs should be modified

pub struct Hello1 {
  //fields
}

pub struct Hello2 {
  //fields
}
