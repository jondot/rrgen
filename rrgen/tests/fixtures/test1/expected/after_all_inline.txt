// Both structs should be modified

pub struct HelloWorld1 {
  //fields
}

pub struct HelloWorld2 {
  //fields
}